### PR TITLE
Develop4

### PR DESCRIPTION
**Pull request by dat26061994**
To day, I edit Member and validate in Server

### CODE REVIEW

File: app/Http/Controllers/MemberController.php [hunk 1]
```diff
@@ -39,15 +39,16 @@ public function getEdit($id)
         return $member->findId($id);
```
Comment (4c0n) [inline app/Http/Controllers/MemberController.php:39]: There is no $id variable declared in this method, but it is used as a parameter to the findId call.

File: public/admin/js/mem.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ app.controller('MemberController', function ($scope, $http, $httpParamSerializer
     }).then(function successCallback(response) {
         $scope.members = response.data;
     }, function errorCallback(response) {
-        alert("Error Show List Member");
+        sweetAlert("Error...", "Something went wrong!", "error");
```
Comment (4c0n) [inline public/admin/js/mem.js:27]: I think you should probably change the error message into something that actually says what went wrong.

File: public/admin/js/mem.js [hunk 3]
```diff
@@ -58,28 +58,25 @@ app.controller('MemberController', function ($scope, $http, $httpParamSerializer
...
+            $('#exampleModaladd').modal('show');
+        }else if(state == 'edit'){
+            $http({
+                method: 'GET',
+                url: 'admin/edit/' + id
+            }).then(function successCallback(response) {
+                $scope.id = id;
+                $scope.file = null;
+                $scope.member = response.data;
+                $('.avatar').attr('src', 'public/upload/' + response.data.avatar);
```
Comment (4c0n) [inline public/admin/js/mem.js:73]: Are you mixing jQuery and Angular here?

File: tests/Feature/BasicTest.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class BasicTest extends TestCase
```
Comment (4c0n) [inline tests/Feature/BasicTest.php:10]: Please try not to commit files like this. It does not do anything.

**Review by 4c0n — CHANGES_REQUESTED** [review]

File: tests/Feature/BasicTest.php (hunk 4 above)
Comment (dat26061994) [reply tests/Feature/BasicTest.php:10]: Sr, i will remember it

File: public/admin/js/mem.js (hunk 3 above)
Comment (dat26061994) [reply public/admin/js/mem.js:73]: Because, i can not use  angular to this show image so, i use Jquery

File: public/admin/js/mem.js (hunk 2 above)
Comment (dat26061994) [reply public/admin/js/mem.js:27]: Thank you , i will fix it in next pull request. Because, i merged pull request before check your comment. Sorry

Comment (4c0n) [reply public/admin/js/mem.js:27]: OK

File: public/admin/js/mem.js (hunk 3 above)
Comment (4c0n) [reply public/admin/js/mem.js:73]: I see

Comment (dat26061994) [reply public/admin/js/mem.js:73]: I will try use AngularJS

**Pull request merged**